deliveries/chainide-polkaholic-integrated-wasm-contract-explorer-integration-milestone_1.md

```diff
@@ -0,0 +1,190 @@
+| Access the WASM contract calls (by Substrate extrinsic hash) |  https://polkaholic.io/tx/`{extrinsicHash}` | [screenshot](https://cdn.polkaholic.io/inkubator-m1/tx-ui.png) [Example](https://shibuya.polkaholic.io/tx/0x4d0461f63da912503c7355db5a79c0e285773c66a383a7d9922b8c4a0852ef33)
+
+This the above can used by any UI (e.g. the ChainIDE, dashboard).  In addition, users can search by the same keys in the Polkaholic.io search bar, e.g.
+* 0x396418ff533172de8407004f53051b5409f6892564ddeba12f75cc855cb16fe0
+* XRcGZzdH841dHySiM62BfJDbQfXXYex3U7LYooRJt8EgJeX
+* 0x4d0461f63da912503c7355db5a79c0e285773c66a383a7d9922b8c4a0852ef33
+
+### Polkaholic.io WASM developer dashboard
+
+The Polkaholic.io WASM developer dashboard was built using [Apache Superset](https://superset.apache.org/), a popular open-source modern data exploration and visualization platform that is hosted at
+
+https://analytics.polkaholic.io
+
+This is connected to Colorful Notion's [substrate-etl](https://github.com/colorfulnotion/substrate-etl) with BigQuery as well as the Polkaholic MySQL replica.  
+
+Many dashboards can be built in Superset very rapidly (with no "front end engineering"), but for Milestone 1 we built two core dashboards, with `{network}` and `{developerAddress}` URL parameters:
+
+|Function|URL|Example|
+|------|----|----|
+| WASM Contract Activity for one network, across all developers | https://analytics.polkaholic.io/superset/dashboard/20/?network=`{network}`&standalone=2 | https://analytics.polkaholic.io/superset/dashboard/20/?network=shibuya&standalone=2 [screenshot](https://cdn.polkaholic.io/inkubator-m1/dashboard20.png)
+| WASM Contract Activity for specific developer on one network |  https://analytics.polkaholic.io/superset/dashboard/57/?network=`{network}`&developer=`{developerAddress}`&standalone=2  |  https://analytics.polkaholic.io/superset/dashboard/57/?network=shibuya&developer=afAEAGz4MVRQjkqUuaUTCU9nTPHruE5xfvazKzPs4kndwso&standalone=2  [screenshot](https://cdn.polkaholic.io/inkubator-m1/dashboard57.png)
+
+ChainIDE integrated both of the above dashboards with IFRAMEs and used the CSS customization capabilities to match its Dark Mode.
+
+It is expected that additional dashboards can be done for other purposes (e.g. a line chart showing # of calls, contracts verified, etc. by day/month/quarter, by network etc.) for other contexts.  The same Superset can be used to expose these in Polkaholic.io, as well support many other blockchain analytics problems.    
+
+## 0c. Testing and Testing Guide
+
+The ChainIDE-Polkaholic unit tests are comprehensively demonstrated in the following video link:
+
+[Comprehensive ChainIDE Walkthrough, including Polkaholic.io integration (Video Link)](https://www.dropbox.com/scl/fi/tf0jgxsmss5b52uk0nbw7/Astar-ChainIDE-M2-New.mp4?rlkey=ts3tpvzvxpbfg7wrl7f4va1yl&dl=0)
```

Comment:
Ya!

One piece of advice here. I'd split the video out into a series of shorter (1-3min) videos to make it easier for people to find what they need quickly.

I'd also like to see this on YouTube or something more accessible

Reply:
This is a good suggestion, and I believe I can have it completed by next week.

Reply:
> Ya!
> 
> One piece of advice here. I'd split the video out into a series of shorter (1-3min) videos to make it easier for people to find what they need quickly.
> 
> I'd also like to see this on YouTube or something more accessible


By the way, do you think using the original video directly is sufficient, or do we need to optimize the original video to meet the specifications for the promotional video?

Reply:
> Ya!
> 
> One piece of advice here. I'd split the video out into a series of shorter (1-3min) videos to make it easier for people to find what they need quickly.
> 
> I'd also like to see this on YouTube or something more accessible

https://www.youtube.com/watch?v=q-Bf5RHSt4s